kafka-streams-cleanup.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Clean-up script to reset a Kafka Streams (v0.10.0.0) application for reprocessing from the very beginning
+#
+# resets topic offsets to zero
+# deletes content of intermediate and result topics (user topics); via kafka-streams-cleanup.sh
+# deletes internal topics completely
+# clears local state store directory
+#
+
+# setup instructions:
+#
+# install librdkafka (required for Kafka Python clients):
+#
+# git clone https://github.com/edenhill/librdkafka
+# cd librdkafka
+# ./configure
+# make
+# sudo make install
+# export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
+#
+# install Kafka Python clients:
+#
+# sudo pip install confluent-kafka
+
+from confluent_kafka import Consumer
+import sys
+import subprocess
+import shutil
+
+def print_usage (appname):
```

Comment:
Why are we including this Python script? It seems out of place (it requires a bunch of things that are not part of Kafka).